orderbook/src/orderbook.rs

```diff
@@ -80,7 +84,9 @@ impl Orderbook {
         if self.unsupported_tokens.contains(&order.sell_token) {
             return Ok(AddOrderResult::UnsupportedToken(order.sell_token));
         }
-        if !has_future_valid_to(shared::time::now_in_epoch_seconds(), &order) {
+        if order.valid_to
+            < shared::time::now_in_epoch_seconds() + self.min_order_validity_period.as_secs() as u32
```

Comment:
We should probably spare a separate error code or rename past valid to to `InusfficientValidTo` to not confuse users of the API.

Reply:
Agree. Changed to that name.